### PR TITLE
[RHDX-165] Add CTA and Embedded Content to Layout: Two Column assembly

### PR DESCRIPTION
**Pull request by ghost**
### JIRA Issue Link
* https://projects.engineering.redhat.com/browse/RHDX-165

### Verification Process
In an effort to ensure the "Eloqua Form" content type can be used in various layouts, it is now added to the "Layout: Two Column" assembly along with the "Embedded Content" assembly.

Sample: https://developer-preview-3532.ext.us-west.dc.preprod.paas.redhat.com/webinars/monolith-to-microservices/edit

To rebuild the current webinar layout being used here: https://developers.redhat.com/webinars/monolith-to-microservices/
1. Login and go to: https://developer-preview-3532.ext.us-west.dc.preprod.paas.redhat.com/webinars/monolith-to-microservices/edit
2. Add a new "Layout: Two Column" section
3. Select the "Right sidebar" Visual Style
4. Create a new "Call to Action" assembly in "Column 1" and build the content
5. Add "AssemblyBackgrounds_RHDBG_dark3.jpg" to the "Background image field
6. Click "Update Section", enter a "Revision log message", and save.

### CODE REVIEW

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/components/_rhd-c-two-col-hero-layout.scss [hunk 1]
```diff
@@ -94,7 +97,9 @@ $pf-global--breakpoint--temp: 1360px;
       padding-left:  var(--rhd-theme--container-spacer-md);
       padding-right:  var(--rhd-theme--container-spacer-md);
     }
+  }
 
+  .field--name-field-column-2 {
```
Comment (staceymosier) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/components/_rhd-c-two-col-hero-layout.scss]: Does this read that `.assembly-type-static_content_band` is only supported in the column 2 field?

Comment (rogeralloymagnetic) [reply _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/components/_rhd-c-two-col-hero-layout.scss]: Yeah, that was what @alistairmc had originally written, I just moved the padding stuff up to apply to both columns. I'm fine with adding the column-1 to that whole block, I'm just not sure if that was done on purpose.

**Review by gdoyle1 — CHANGES_REQUESTED** [review]
@rogeralloymagnetic  Need a couple changes for responsive views:
Screen Shot 2020-03-24 at 3 09 07 PM
1. "Sidebar" needs to fall below the larger assembly (aka rich text in this case)
2. CTA needs to be left-aligned with the form
3. Form needs to be flushed all the way left so that it matches with where the text starts on the page 

Screen Shot 2020-03-24 at 3 09 16 PM
1. Form needs to be flushed all the way left so that it matches with where the text starts on the page

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/components/_rhd-c-two-col-hero-layout.scss (hunk 1 above)
Comment (alistairmc) [reply _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/components/_rhd-c-two-col-hero-layout.scss]: If I remember, it was to make the nested components work. happy if it's been tested to work ok

**Comment by rogeralloymagnetic** [comment]
@gdoyle1 The last push should resolve all the issues you saw, but let me know if you have any other tweaks.

----------------------------------- Desktop Columns -----------------------------------
Screen Shot 2020-03-25 at 12 39 55 PM

----------------------------------- Mobile Columns -----------------------------------
Screen Shot 2020-03-25 at 12 40 09 PM

----------------------------------- Form Padding -----------------------------------
Screen Shot 2020-03-25 at 12 42 17 PM

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/components/_rhd-c-two-col-hero-layout.scss (hunk 1 above)
Comment (rogeralloymagnetic) [reply _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/components/_rhd-c-two-col-hero-layout.scss]: @staceymosier resolved with #428f050

**Review by gdoyle1 — APPROVED** [review]
Let's merge it! @rogeralloymagnetic

**Review by alistairmc — APPROVED** [review]
This looks great now

**Review by staceymosier — APPROVED** [review]
Checked out the yml and went through the admin -- Looks good!

**Pull request merged**